lib/internal/modules/esm/utils.js

```diff
@@ -97,9 +99,68 @@ function initializeESM() {
   setImportModuleDynamicallyCallback(importModuleDynamicallyCallback);
 }
 
+async function initializeHooks() {
+  const customLoaderPaths = getOptionValue('--experimental-loader');
+
+  let cwd;
+  try {
+    cwd = process.cwd() + '/';
```

Comment:
I couldn't find any documentation regarding `process.cwd()` throwing an error. Is this correct?

Reply:
See https://github.com/nodejs/node/blob/73c0564fcaea8dba8b480fefec4e9f1099a9cb91/src/node_process_methods.cc#L132-L134

Reply:
That doesn't explain to much though 😅

I asked about this previously: https://github.com/nodejs/node/pull/37468#discussion_r624739134

The answer is: https://github.com/nodejs/node/pull/37468#discussion_r691388910

> this can fail if the parent directory is deleted while the process runs